requirements/dev.txt

```diff
@@ -6,7 +6,7 @@ pytest==7.4.*
 coverage==7.4.*
 pytest-timeout==2.2.*
 pytest-mock==3.12.*
-onnx==1.15.0
+onnx==1.16.0
```

Comment:
Just a question. Do we need this in dev.txt? It's already in openvino.txt

Reply:
I don't know why it's in the dev.txt